app/web/package.json

```diff
@@ -71,25 +71,23 @@
     "@storybook/addon-essentials": "^6.5.10",
     "@storybook/addon-links": "^6.5.10",
     "@storybook/builder-webpack5": "^6.5.10",
-    "@storybook/cli": "^8.4.5",
     "@storybook/manager-webpack5": "^6.5.10",
     "@storybook/react": "^6.5.10",
+    "@testing-library/dom": "7.21.4",
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^12.1.5",
     "@testing-library/react-hooks": "^8.0.1",
     "@testing-library/user-event": "^13.4.1",
     "@types/css-mediaquery": "^0.1.4",
     "@types/geojson": "^7946.0.10",
     "@types/google-protobuf": "^3.15.12",
-    "@types/jest": "^28.1.6",
     "@types/markdown-it": "^14.1.2",
     "@types/node": "^22.10.1",
     "@types/react": "^17.0.43",
-    "@types/react-dom": "^17.0.13",
```

Comment:
unused

Reply:
Hmm wouldn't this cause an error if we try to use anything from the react-dom package (e.g. createPortal)

Reply:
oh wait looks like the diff I commented on is outdated? I can still see it in the final diff 🤔

Reply:
Oh yeah whoops I put it back, I thought it got rid of the comment, sorry!